### ISSUE TITLE
jupyter notebook list gives port error

### ISSUE THREAD

**Issue by michaelaye** [jupyter/notebook#717]
This might be just a beauty problem, but it is confusing to get a port error when all I want is a list of all running notebook servers that obviously will include different ports:

```bash
$ jupyter notebook list
[I 09:01:36.652 NotebookApp] The port 8888 is already in use, trying another random port.
Currently running servers:
http://localhost:8888/ :: /Users/klay6683/Dropbox/src/planet4/notebooks
```
jupyter notebook version: 4.0.6

**Comment by minrk** [comment]
I think this is fixed in master (4.1)

**Comment by willingc** [comment]
@minrk Verified. Fixed in 4.1.0b1.
<img width="901" alt="screen shot 2015-12-16 at 5 36 00 am" src="https://cloud.githubusercontent.com/assets/2680980/11842249/579bd310-a3b7-11e5-9260-6a9155074590.png">
